pyvista/_plot.py

```diff
@@ -200,9 +200,7 @@ def plot(
     ImageData. Note ``volume=True`` is passed.
 
     >>> import numpy as np
-    >>> grid = pv.ImageData(
-    ...     dimensions=(32, 32, 32), spacing=(0.5, 0.5, 0.5)
-    ... )
+    >>> grid = pv.ImageData(dimensions=(32, 32, 32), spacing=(0.5, 0.5, 0.5))
```

Comment:
Lines are wrapped more consistently than with blackdoc (length is 69 < limit of 75)